### PR TITLE
MOTECH-2211: Documentation for IE9 issues

### PR DESCRIPTION
**Pull request by sebastianceronik**
added information about IE9 inabilities to handle 3 digit hexes, filter gradient properties, and incompatibility with css3 animations and transitions

### CODE REVIEW

File: docs/source/development/browser.rst [hunk 1]
```diff
@@ -34,21 +34,36 @@ AngularJS 1.3 has dropped support for IE8. Read more about it on this `blog <htt
 
 The AngularJS project currently supports and will attempt to fix bugs for IE9 and above. Their continuous integration server runs all the tests against IE9, IE10, and IE11. See `Travis CI <https://travis-ci.org/angular/angular.js>`_ and `ci.angularjs.org <https://ci.angularjs.org>`_.
 
+Internet Explorer 9: Issues
+---------------------------
+
+To keep Compatibility with Internet Explorer 9 it is best to follow these rules:
+
+1. Internet Explorer 9 don't support  CSS3 animations and transitions so it is best to use *.gif files for animations.
```
Comment (pgesek) [inline docs/source/development/browser.rst]: doesn't

Comment (pgesek) [inline docs/source/development/browser.rst]: remove double space before CSS3

Comment (pgesek) [inline docs/source/development/browser.rst]: add comma before "so"

File: docs/source/development/browser.rst [hunk 2]
```diff
@@ -34,21 +34,36 @@ AngularJS 1.3 has dropped support for IE8. Read more about it on this `blog <htt
...
+2. Internet Explorer 9 filter support only colours defined in 8 digit hexes (AARRGGBB)
+
+    .. code-block:: bash
+
+			filter : progid:DXImageTransform.Microsoft.gradient(startColorstr='#FFE8E8E8', endColorstr='#FFF2F2F1');
+
+   For more information see `Gradient Attributes and Properties <https://msdn.microsoft.com/en-us/library/ms532997(v=vs.85).aspx>`_
+   Some frameworks don't use correctly defined colours for MS filter and it might lead to inconsistency between different browsers UI.
```
Comment (pgesek) [inline docs/source/development/browser.rst]: browser UIs

**Comment by kmadej** [comment]
PR closed because of open new one. More information in https://github.com/motech/motech/pull/416

**Pull request closed without merging**